manifests/base/certificate/certificate.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
There has been some discussions about making cert-manager opt-in triggered by kubeflow/trainer#2426 and captured in kubeflow/manifests#3002, so we may want to move cert-manager manifests in the `third-party/cert-manager` directory instead of `base/certificate`.